compiler/modules/front/codeGen.bal

```diff
@@ -1025,8 +1025,9 @@ function codeGenExpr(CodeGenContext cx, bir:BasicBlock bb, Environment env, s:Ex
             return codeGenTypeTest(cx, bb, env, td, left, negated);
         }
         // Variable reference
-        var { varName } => {
-            return codeGenVarRefExpr(cx, varName, env, bb);
+        var { varName, prefix } => {
```

Comment:
@jclark This doesn't match after adding also getting below at compile time
`WARNING [codeGen.bal:(1028:9,1028:32)] pattern will not be matched`
Is this a bug or am I missing something ?

Reply:
`match` seems to have lots of bugs

Reply:
` var ref if ref is s:VarRefExpr => {` works as a workaround.